woocommerce/woocommerce-gutenberg-products-block
# E2E Next Previous block

This PR introduced E2E tests for #10235. 

At the beginning, I tried to implement a visual comparison but I didn't like the approach. However, I kept the change to the playwright configuration given that it could be useful in the future (it is just one line). 

Testing instructions:

1. `nvm use`
2. `npm install`
3. `npm run env:start`
4. `npm run test:e2e tests/e2e/tests/product-gallery/inner-blocks/product-gallery-large-image-next-previous/product-gallery-large-image-next-previous.block_theme.spec.ts`
5. Make sure all tests pass.
